setup.py

```diff
@@ -93,9 +93,9 @@ def _get_readme():
                         'passlib>=1.6.2,<2.0',
                         'pexpect>=3.3,<4.0',
                         'requests>=2.4.1,<3.0',
-                        'pytest>=2.6.2,<3.0',
+                        'pytest>=2.6.2,<4.0',
```

Comment:
Please see Ian and I's discussion in the comments. We should be updating these all the time.